src/components/SharedComponents/UploadStatus/UploadStatus.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+
+import INatIcon from "components/INatIcon";
+import { View } from "components/styledComponents";
+import * as React from "react";
+import CircularProgressBase from "react-native-circular-progress-indicator";
+import colors from "styles/tailwindColors";
+
+type Props = {
+  color: string,
+  completeColor: string,
+  progress: number
+}
+
+const UploadStatus = ( { color, completeColor, progress }: Props ): React.Node => (
```

Comment:
Please also give the color and completeColor a default value of darkGray.

Reply:
Should the default value for completeColor be inatGreen instead?

Reply:
Yes, probably better so.